lib/src/client.dart

```diff
@@ -2561,16 +2561,16 @@ class Client extends MatrixApi {
       if (room != null) {
         final List<BasicEvent> events = [];
         for (final event in _eventsPendingDecryption) {
-          if (event.event.roomID != roomId) continue;
+          if (event.event.room.id != roomId) continue;
           if (!sessionIds.contains(
-            event.event.content['content']?['session_id'],
+            event.event.content.tryGet<String>('session_id'),
           )) {
             continue;
           }
 
-          final decryptedEvent = await event.event.decrypt(room);
-          if (decryptedEvent.content.tryGet<String>('type') !=
-              EventTypes.Encrypted) {
+          final decryptedEvent =
+              await encryption!.decryptRoomEvent(event.event);
```

Comment:
I have a doubt, do we use `EventUpdateType.timeline` for all to-device events?

Reply:
How to you mean this? 🤔

Reply:
I was thinking if we should have a seperate enum value for to-device events. Like `EventUpdateType.toDevice` but handle it the same way as `timeline`. Just a nitpick though.

Not a blocker so I'll approve this one :)

Reply:
I just thought it would make it obvious that it's different from usual events when logging and makes code a little easy to understand

Reply:
> I was thinking if we should have a seperate enum value for to-device events. Like EventUpdateType.toDevice but handle it the same way as timeline. Just a nitpick though.

Ah okay. We don't handle to-device messages as EventUpdates. Only events which are tied to a room are handled here